resources/sass/aante-dark.scss

```diff
@@ -1,23 +1,24 @@
 // $author Ante Aljinovic https://github.com/aljinovic
 // modified dark version
 
-$main-background: invert(#f8f8f8);
-$secondary-background: invert(#f8f8f8);
+@import "aante";
 
-$variable-type-color: invert(#06f);
-$variable-type-color-hover: invert(#f00);
+:root {
+  --main-background: #070707;
+  --secondary-background: #070707;
```

Comment:
I wouldn't say a "technical" reason, but here are the reasons:
1. `invert(...)` is a Sass feature, not native to CSS. I could not figure out how to apply the colors so that it would transpile correctly.
2. In the upcoming commits, I plan to "refresh" the colors that is used. When @LoranRendel helped add `aante-dark` last year, they used the inverted colors, which is fine, but if we used "dark-optimized" values (that are not just the `invert()`-ed variant of the light ones), it can look better.

This means that if the chosen values are completely unrelated to those of the light variant, we can forgo using `invert()` altogether and just use custom dark-optimized values that look nicer.